environments/workspaces-as-code/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Workspaces as code"
+description: "Learn how to describe environment configuration as code."
+state: beta
+---
+
+Workspaces as code (WAC) brings the _Infrastructure As Code_ paradigm to Coder
+environments. WAC allows you to define and create new environments using
+**workspace templates**.
```

Comment:
@khorne3 should this be a proper noun?

Reply:
No, we're not capitalizing any feature names (just the names of products)